app/build.gradle

```diff
@@ -43,18 +43,17 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_17
-        targetCompatibility = JavaVersion.VERSION_17
+        sourceCompatibility = JavaVersion.VERSION_11
+        targetCompatibility = JavaVersion.VERSION_11
     }
 
     kotlinOptions {
-        jvmTarget = "17"
+        jvmTarget = "11"
     }
     namespace 'com.google.maps.android.ktx.demo'
 }
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
Wasn't used and AFAIK is considered a bad practice because makes the build less deterministic